tests/test_lab_dev/test_states.py

```diff
@@ -769,6 +770,57 @@ def test_representation_error(self):
             SqueezedVacuum(modes=[0], r=[0.1, 0.2]).representation
 
 
+class TestTwoModeSqueezedVacuum:
```

Comment:
there are some other nice tests, like that if you trace one subsystem e.g. using `state[0]` you should get a thermal state.